Update install-provisioningpackage.md (#1190)
MicrosoftDocs/windows-powershell-docs: docset/windows/provisioning/install-provisioningpackage.md

```

## DESCRIPTION

> This cmdlet is used to instlled exported .ppkg files that can be generated and exported from the [Windows Configuration Designer Tool](https://docs.microsoft.com/en-us/windows/configuration/provisioning-packages/provisioning-install-icd).

Reviewer: ```suggestion
This cmdlet is used to install .ppkg files that are generated and exported by the [Windows Configuration Designer tool](https://docs.microsoft.com/windows/configuration/provisioning-packages/provisioning-install-icd).
```

> You can direct this cmdlet to install a .ppkg file interactively or silently by specifying the /quiet install switch.

Reviewer 2: ```suggestion
You can use this cmdlet to install a .ppkg file interactively or silently by specifying the -QuietInstall switch parameter.
```

> You can direct this cmdlet to install a PPKG file interactively or silently by specifying the -QuietInstall Switch.

Reviewer: ```suggestion
You can direct this cmdlet to install a .ppkg file interactively or silently by specifying the -QuietInstall Switch.
```

Reviewer: Please check technical accuracy for this edit.

> The default is an interactive install

Reviewer: ```suggestion
The default is an interactive install.
```

## EXAMPLES

### Example 1
```
PS C:\> Install-ProvisioningPackage -PackagePath C:\mypackage.ppkg -QuietInstall  
```

> The snippet above will install the mypackage.ppkg file quietly without user interaction.

Reviewer: ```suggestion
This will install the mypackage.ppkg file silently without user interaction.
```

```

### -QuietInstall

> Quietly installs the package with no end user interaction

Reviewer: ```suggestion
Quietly installs the package without any customer interaction.
```
